config/jobs/kubernetes-sigs/lws/lws-presubmit.yaml

```diff
@@ -53,3 +53,33 @@ presubmits:
           requests:
             cpu: 3
             memory: 10Gi
+  - name: pull-lws-test-e2e-main
```

Comment:
Let's follow kueue/jobset lead on this.

We should have e2e tests for the supported versions of kubernetes (and please title them with the k8s version in the name).

So as of writing, this would be 1.27, 1.28, and 1.29.

Reply:
Looks like you addressed this.